.changeset/light-turtles-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Removed deprecated types that are exported in plugins/catalog-backend/src/ingestion/index.ts. Also fixed deprecated use of "substr" into "substring" in the files plugins/catalog-backend/src/modules/core/PlaceholderProcessor.ts and plugins/catalog-backend/src/service/request/parseEntityFilterParams.ts.
```

Comment:
if you put words in backticks then vale will ignore it 😄

Reply:
Okay, so I should remove the " -symbol and then it won't ignore it?

Reply:
you can put it in ` instead of "